visual_tests.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```

Comment:
That's handy.  There's also `tools/test_triage.py` that does something similar in PyQt4 in case you're not aware of it.